src/cmake-tools.ts

```diff
@@ -2152,7 +2156,15 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
         }
     });
 
-    private _createTerminal(options: vscode.TerminalOptions, executable: api.ExecutableTarget) {
+    private async createTerminal(executable: api.ExecutableTarget): Promise<vscode.Terminal> {
+        const user_config = this.workspaceContext.config.debugConfig;
+        const drv = await this.getCMakeDriverInstance();
```

Comment:
Did this code need to move? If you are moving it, I recommend putting it closer to where it's needed (line 2190) since an early return on line 2185 could mean that we don't need to initialize it at all.  It also means you don't need to check if `options` is defined or not.